meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainerAuto.sh

```
@@ -11,103 +17,30 @@ killall  -9 jsdelivr-systemWatchdog.sh
-    killall jsdelivr-keepWatchdogHappy.sh
-    exit 1
-fi
-
-SERVER_VER=`curl --silent https://data.jsdelivr.com/v1/packages/gh/jsdelivr/globalping-probe/resolved   | awk -F ':'  '/"version"/ {print  substr($NF, 1, length($NF)-1)}'`
-CURRENT_VER=`cat /tmp/AutoUpdate/CURRENT_VERSION`
-
-echo "VERSIONS:" > /dev/tty5
-echo "SERVER_VERSION: $SERVER_VER" > /dev/tty5
-echo "CURRENT_VERSION: $CURRENT_VER" > /dev/tty5
+CONTAINERS=$(docker ps -aq 2>/dev/null)
+[ -n "$CONTAINERS" ] && docker stop $CONTAINERS
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)